### DISCUSSION TITLE
Using a quadrat instead of lasers, and video overlay

### DISCUSSION THREAD

**Discussion by LLTeed** [biigle/core#535] [Q&A] [answered]
Hi there,

I've just started using BIIGLE and I will be analyzing imagery that has visible quadrats in each image, not lasers. Just wondering how to go about this for my area calculations, if I just label the quadrat edges as laser points and put the distance as 25cm?

Also, for another project, I will be working with imagery/video that may contain an overlay (we are still trying to decide camera specs). Is it possible to exclude the overlay from the area calculations (i.e., draw a box within the overlay to use)? Or would you suggest removing the overlay?

Thank you in advance!

**Comment by mzur** [comment, marked as answer]
> I've just started using BIIGLE and I will be analyzing imagery that has visible quadrats in each image, not lasers. Just wondering how to go about this for my area calculations, if I just label the quadrat edges as laser points and put the distance as 25cm?

Yes, BIIGLE [supports](https://biigle.de/manual/tutorials/laserpoints/laserpoint-detection) the area calculation of a square and it doesn't matter if it's laser points or the corners of your quadrat. It just doesn't work with the automatic laser point detection, so you have to annotate all the corners manually.

> Also, for another project, I will be working with imagery/video that may contain an overlay (we are still trying to decide camera specs). Is it possible to exclude the overlay from the area calculations (i.e., draw a box within the overlay to use)? Or would you suggest removing the overlay?

Could you please give an example how this overlay would look like? Do you mean metadata like a timestamp or the camera depth? These should not matter for the area calculation as long as the overlay does not hide the reference points (i.e. laser points or corners of a quadrat). In the image annotation tool you can also define an "export area" that is constantly shown. When you generate a new report, you can limit the report to annotations within the export area only.

The video annotation tool currently does not support area calculations or an export area.

There is also a feature idea to display the image/video metadata in an overlay that is configurable in BIIGLE ( https://github.com/biigle/core/issues/462 ). This would allow you to show or hide metadata fields on-demand.

**Reply by LLTeed** [reply]
Great, thank you for clarifying! In our test runs, the overlay looks like the attached image. We are hoping to have date/time, transect ID, depth, altitude, lat and long as an overlay.
![CON_003_1 4m_140708](https://user-images.githubusercontent.com/104437250/212699517-69939e42-1ad9-4071-ba4e-d53bd6f034a5.png)

**Reply by mzur** [reply]
I see. The overlay will be ignored for the area calculations as long as it does not hide the reference points (i.e. laser points or corners of a quadrat). **But...** please be aware that the area calculation is not accurate for oblique angle imagery. It assumes a more or less flat underground and a camera that looks straight down. Even then the accuracy of the calculated area decreases with a higher distance to the reference points (depending on the optics of the camera which are not taken into account).

**Reply by LLTeed** [reply]
Okay, great thank you! Looking at #462 I was just wondering what file formats are supported to import overlays as metadata?

**Reply by mzur** [reply]
You can import metadata in the BIIGLE CSV format or iFDO (see the [manual](https://biigle.de/manual/tutorials/volumes/file-metadata)). It's just that video metadata is currently not used anywhere.

**Reply by LLTeed** [reply]
Great, thank you for your help!
